analyzer/javacg-opal/src/test/java/eu/fasten/analyzer/javacgopal/data/OPALCallGraphConstructorIntegrationTest.java

```diff
@@ -12,6 +12,7 @@
 import org.apache.commons.io.FileUtils;
 import org.apache.commons.lang3.builder.EqualsBuilder;
 import org.apache.commons.lang3.builder.HashCodeBuilder;
+import org.junit.jupiter.api.Disabled;
```

Comment:
Why is there a new import, but no use of the annotation in the file?

Reply:
fixed